access/templates/access/graded_form.html

```diff
@@ -138,6 +142,9 @@
 
 			{% else %}
 			<!-- Text-field -->
+			<div class="{{ field.field.type }}">
+				{{ field.tag }}
```

Comment:
Hmm.. I guess this will work with all different text input types, which seem to be selected based on the comparison method). If not, then `{{ field }}` might work.